bin/export_open_targets.sh

```diff
@@ -9,6 +9,10 @@ destination=${destination:-"$ATLAS_FTP/experiments/cttv010-$(date "+%Y-%m-%d").j
 usageMessage="Usage: (-a $atlasUrl) (-p urlParams:$urlParams) (-d destination:$destination) (-o outputPath:outputPath)"
 venvPath=${venvPath:-"$ATLAS_PROD/venvs"}
 
+[ ! -z ${opentargetsValidator+x} ] || ( echo "Env var opentargetsValidator version needs to be defined." && exit 1 )
```

Comment:
```suggestion
[ ! -z ${opentargetsValidatorVersion+x} ] || ( echo "Env var opentargetsValidatorVersion needs to be defined." && exit 1 )
```
so that we are explicit on what is the var about.